mlflow/genai/scorers/base.py

```diff
@@ -380,14 +380,15 @@ def _reconstruct_decorator_scorer(cls, serialized: SerializedScorer) -> "Scorer"
         object.__setattr__(scorer_instance, "_cached_dump", original_serialized_data)
         return scorer_instance
 
-    def run(self, *, inputs=None, outputs=None, expectations=None, trace=None):
+    def run(self, *, inputs=None, outputs=None, expectations=None, trace=None, session=None):
```

Comment:
@xsh310 Can you confirm whether this change makes sense in light of your other changes to the `Scorer` class? I noticed that you updated `__call__` but not `run`.